lib/src/modules/instabug.dart

```diff
@@ -182,11 +183,12 @@ class Instabug {
   }) async {
     $setup();
     InstabugLogger.I.logLevel = debugLogsLevel;
-    return _host.init(
+    await _host.init(
       token,
       invocationEvents.mapToString(),
       debugLogsLevel.toString(),
     );
+    FeatureFlagsManager.registerW3CFlagsListener();
```

Comment:
Should we return the `Future` returned from `registerW3CFlagsListener` here for users to be able to `await` till `init` is done?